tests/e2e/framework/framework.go

```diff
@@ -203,7 +203,7 @@ func (c *CommonConfig) RunTest(m runnable) int {
 		ret = m.Run()
 	}
 	if err := c.saveLogs(ret); err != nil {
-		glog.Warning("Log saving inconplete: %s", err)
+		glog.Warning("Log saving incomplete")
```

Comment:
Or you can drop the err, if you think it does not help. I think either one is fine.